node_modules/col-analytics/lib/caliper.js

```diff
@@ -58,19 +63,185 @@ var init = module.exports.init = function() {
  * @return {void}
  */
 var track = module.exports.track = function(user, event, metadata) {
-  try {
-    getCourseIfMissing(user, function(err, course) {
-      if (err) {
-        log.error({'user': user}, 'Could not retrieve course for user');
+  getCourseIfMissing(user, function(err, course) {
+    if (err) {
+      log.error({'user': user}, 'Could not retrieve course for user');
```

Comment:
@paulkerschen `user.id` instead of `user` above?  We don't get JSON.stringify in prod, right?